Document/Document.ipynb

```diff
@@ -2127,6 +2127,20 @@
     "<center><p style=\"text-align:center;\">Fig2. Parameter Recommender Block Diagram</p></center>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "$$R=\\frac{Max(P)}{Min(P)}$$"
```

Comment:
I think it would be better if we provide a short description before writing the formulation e.g. "For determining if the dataset is imbalanced, we use the following strategy:"

Reply:
![](https://www.memecreator.org/static/images/memes/4074349.jpg)